system-test/bigtable.ts

```diff
@@ -698,6 +698,25 @@ describe('Bigtable', () => {
         await TABLE.insert(rows);
       });
 
+      it('should include metadata in error sent back', async () => {
+        const rows = [
+          {
+            key: 'gwashington',
+            data: {
+              nonExistentColumnFamily: {
+                jadams: 1,
+              },
+            },
+          },
+        ];
+        try {
+          await TABLE.insert(rows);
+          assert.fail('Inserts should not work on non-existent column family.');
```

Comment:
`assert.fail` is always going to be throwing an `AssertionError` thus always triggering the following `catch` block

```suggestion
          // Inserts should not work on non-existent column family.
          await TABLE.insert(rows);
```

That message will live better as a comment or something similar

Reply:
Are you sure? I want the test to fail if the insert succeeds. In this case it will go to the catch block, but then the other assertion will fail.